src/main/java/ch/njol/skript/conditions/CondIsOnline.java

```diff
@@ -33,32 +33,41 @@
  * @author Peter Güttinger
  */
```

Comment:
We able to remove the author comment?

Reply:
No that should be done in a batch change rather than messing around individually.